dotty-tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/api/keras/layers/LSTM.scala

```diff
@@ -0,0 +1,7 @@
+package me.shadaj.scalapy.tensorflow.api.keras.layers
+
+import me.shadaj.scalapy.tensorflow.keras.layers.{LSTM => PyLSTM}
+import me.shadaj.scalapy.tensorflow.api.scalaUtils.PythonUnion._
+import me.shadaj.scalapy.tensorflow.api.scalaUtils.PythonOption._
+
+class LSTM private[api] (override val underlying: PyLSTM) extends Layer(underlying)
```

Comment:
Maybe add at least a scaladoc with the expansion of `LSTM` so that the poor laymen don't need to google it up ;D

Reply:
Long short-term memory (LSTM) sure